modules/Sources/Features/BalanceBreakdown/BalanceBreakdownView.swift

```diff
@@ -230,7 +230,7 @@ extension BalanceBreakdownView {
             .padding(.bottom, 15)
             .disabled(!viewStore.isShieldableBalanceAvailable || viewStore.isShieldingFunds)
             
-            Text(ZatoshiStringRepresentation.feeFormat)
+            Text("(\(ZatoshiStringRepresentation.feeFormat))")
```

Comment:
Is this a good approach to move the brackets into the UI layer instead of keeping them as part of the translatable text?

Reply:
With 2 different instance of Typical Fee in the UI, I decided to have just 1 core translated fee. The parenthesis are general so whenever some screen/UI needs to wrap it inside (), it's done on that specific level.

Reply:
I see, but in this particular case, we have the parenthesis part of the UI, and thus not translated